Remotion/Web/Development.WebTesting/Accessibility/AccessibilityAnalyzer.cs

```diff
@@ -140,59 +142,68 @@ public void IgnoreRule (AccessibilityRuleID rule)
     /// Analyzes specific <see cref="ControlObject"/> on Webpage.
     /// </summary>
     /// <param name="controlObject"><see cref="ControlObject"/> to analyze.</param>
+    /// <param name="timeout">
+    /// Maximum wait time after which an iframe on the page must be visible to be analyzed. If <see langword="null"/>, a default value of 3000 ms is used.
+    /// </param>
     /// <returns> Result of the analysis.</returns>
     [NotNull]
-    public AccessibilityResult Analyze ([NotNull] ControlObject controlObject)
+    public AccessibilityResult Analyze ([NotNull] ControlObject controlObject, [CanBeNull] TimeSpan? timeout = null)
     {
       ArgumentUtility.CheckNotNull ("controlObject", controlObject);
 
-      return Analyze ($"#{controlObject.GetHtmlID()}");
+      return Analyze ($"#{controlObject.GetHtmlID()}", timeout);
     }
 
     /// <summary>
     /// Analyzes a specific element on webpage.
     /// </summary>
     /// <param name="cssSelector">CSS selector of the <see cref="ControlObject"/> to analyze.</param>
+    /// <param name="timeout">
+    /// Maximum wait time after which an iframe on the page must be visible to be analyzed. If <see langword="null"/>, a default value of 3000 ms is used.
+    /// </param>
     /// <returns> Result of the analysis.</returns>
     [NotNull]
-    public AccessibilityResult Analyze ([NotNull] string cssSelector)
+    public AccessibilityResult Analyze ([NotNull] string cssSelector, [CanBeNull] TimeSpan? timeout = null)
```

Comment:
Redundant on a nullable value type.

```suggestion
    public AccessibilityResult Analyze ([NotNull] string cssSelector, TimeSpan? timeout = null)
```

Reply:
good eye. missed that one on develop :o